packages/main/src/SelectPopover.hbs

```diff
@@ -21,8 +21,23 @@
 				>
 				</ui5-button>
 			</div>
+			{{#if _isPhone}}
+				{{#if hasValueStateText}}
+					<div class="{{classes.popoverValueState}} row" style="{{styles.dialogHeader}}">
```

Comment:
Is this "row" class real or a leftover?

Reply:
This is real class which comes from ui5-resposnive-popover. It is used here to set correct font-size and other things to align the value state message to dialog's styles on mobile.